scripts/backend_test_shards.json

```diff
@@ -3,6 +3,7 @@
         "core.controllers.base_test",
         "core.controllers.collection_editor_test",
         "core.controllers.editor_test",
+        "core.controllers.learner_goals_test",
```

Comment:
For load balancing, can this be moved to shards 4 or 5?

Reply:
Done.